examples/loader-duckdb/README.md

```diff
@@ -0,0 +1,9 @@
+[Framework examples →](../)
+
+# DuckDB data loader
+
+View live: <https://observablehq.observablehq.cloud/framework-example-loader-duckdb/>
+
+This Observable Framework example demonstrates how to write a data loader as a bash script that downloads a CSV file from Eurostat (the data portal from the statistical office of the European Union), then calls DuckDB to filter and reshape this data, saving the result as Apache Parquet. The example illustrates the technique by fetching statistics about education to modern foreign languages across Europe.
```

Comment:
```suggestion
This Observable Framework example demonstrates how to write a data loader as a shell script that downloads a CSV file, then uses the DuckDB CLI to filter and reshape this data, finally outputting the result as Apache Parquet. The example data is statistics about the education of modern foreign languages from Eurostat, the data portal of the statistical office of the European Union.
```

`sh` and Bash are [not technically the same thing…](https://stackoverflow.com/questions/5725296/difference-between-sh-and-bash) so better to call this a shell script. Also edited for clarity (and we care less about the specific dataset than the demonstrated technique).